src/HydrationRepository.js

```diff
@@ -0,0 +1,14 @@
+import Hydration from "./Hydration";
+
+class HydrationRepository {
+  constructor(rawHydrationData) {
+    this.hydrationData = this.instantiateHydroData(rawHydrationData);
+  }
+
+  instantiateHydroData(rawHydrationData) {
+    return rawHydrationData.map((data) => new Hydration(data));
+  }
+}
+
+
+export default HydrationRepository;
```

Comment:
Thanks for creating this new class. It'll make it easier to manage the raw data.